src/encoded/static/components/app.js

```diff
@@ -430,6 +430,8 @@ export default class App extends React.Component {
             // We need to rebuild tooltips after navigation to a different page.
             ReactTooltip.rebuild();
```

Comment:
Remove `ReactTooltip.rebuild();` call since `App.debouncedOnNavigationTooltipRebuild();` below should handle it (no point doing 2x in rapid succession).